bin/citrea/provers/risc0/guest-bitcoin/Cargo.lock

```diff
@@ -765,6 +781,27 @@ dependencies = [
  "syn_derive",
 ]
 
+[[package]]
+name = "brotli"
```

Comment:
We dont do brotli in batch prover

Reply:
We will

Reply:
Then we will add this dep when we need it

Reply:
well actually for batch prover we don't need it that's correct.

how to remove it?

Reply:
it's coming from primitives, and also bitcoin da verifier now needs it, need a lot of feature gating to prevent this I  guess

Reply:
I think its due to bitcoin-da/helpers/compression.rs file. its used in batch proof zk, so its not guarded behind native. but this causes light client to take this dependency as well. maybe we can add a compression feature, and native would automatically inherit that. but for zk you have opt-in specifically.